### PR TITLE
Bump dependencies to latest version

### PR DESCRIPTION
**Pull request by bradymholt**
Bump dependencies to latest versions.  As part of the bump, vue-loader had to be included as a plugin in the webpack config.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,24 +5,24 @@
...
+    "npm-check-updates": "^2.14.2",
+    "vue": "2.5.16",
+    "vue-loader": "^15.2.4",
+    "vue-server-renderer": "2.5.16",
+    "vue-template-compiler": "2.5.16",
+    "webpack": "^4.16.0",
+    "webpack-cli": "^3.0.8",
+    "webpack-dev-server": "^3.1.4",
+    "xhr": "^2.5.0",
+    "ynab": "^1.1.0"
```
Comment (shama) [inline package.json]: We need `chalk` and `cpr` in the dependencies in order for `npx ynab-api-starter-kit` to continue working. The other ones I think are ok to move out.

File: webpack.config.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 const path = require('path');
 const webpack = require('webpack');
+const { VueLoaderPlugin } = require("vue-loader");
+
 module.exports = {
   entry: './src/main.js',
+  mode: "development",
```
Comment (shama) [inline webpack.config.js]: We could use `process.env.NODE_ENV || 'development'` for the `mode` and then we don't need to additionally supply it as a parameter in the build script.

File: package.json (hunk 1 above)
Comment (bradymholt) [reply package.json]: Thanks - reverted here: fdad4a621a236bfb8a38fb86baeb0f9a239ac004

File: webpack.config.js (hunk 2 above)
Comment (bradymholt) [reply webpack.config.js]: Much better.  Changed here: 2a88f3f139397f3180f28466a3368f4a83c29751

**Comment by bradymholt** [comment]
@shama You 👍 with the latest changes here?

**Pull request merged**